script.js

```diff
@@ -34,10 +34,17 @@ const addItemToList = () => {
 listContainer.addEventListener("click", (e) => {
   if(e.target.tagName.toLowerCase() === deleteItemTag) {
     e.target.parentElement.remove();
-    storeData()
   } else if (e.target.tagName.toLowerCase() === inputItemTag) {
     e.target.toggleAttribute('checked');
-    storeData();
+  }
+  storeData();
+});
+
+window.addEventListener("keydown", (e) => {
```

Comment:
Let's add the listener on the direct target (`inputField`) instead of `window`.

As far as our app grows we will add other inputs and there would be some undesirable collisions and bugs on it. Let's consider such a case, you have some second input and you press `esc` - what's going to happen? Your `inputField` will be cleared as well, but in this case it isn't supposed to happen, is it?

Reply:
Yes, it is) Agree. Fixed